src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/ExpectedConfigurations/AuthenticationRedacted.json

```diff
@@ -1,6 +1,7 @@
 {
-  "MonitorApiKey": {
-    "Subject": "ae5473b6-8dad-498d-b915-ffffffffffff",
-    "PublicKey": ":REDACTED:"
-  }
+    "MonitorApiKey": {
+        "Subject": "ae5473b6-8dad-498d-b915-ffffffffffff",
+        "PublicKey": ":REDACTED:"
+    },
+    "AzureAd": ":NOTPRESENT:"
```

Comment:
nit: I'm assuming this is sufficient to get the tests to pass, but you could consider having a configuration where `AzureAd` is present (similar to what is done above for `MonitorApiKey`).